RedCanary ingesting the same incident - bug fix (#8237)
demisto/content: Packs/RedCanary/ReleaseNotes/1_0_3.md

#### Integrations
##### Red Canary

> - Fixed an issue in which the same detection was fetched multiple times.

Reviewer: ```suggestion
- Fixed an issue where the same detection was fetched multiple times.
```
